android/src/test/java/com/onegini/mobile/sdk/reactnative/StartClientUseCaseTests.kt

```diff
@@ -86,8 +86,8 @@ class StartClientUseCaseTests {
         }
 
         StartClientUseCase(oneginiSdk, reactApplicationContext)(TestData.config, promiseMock)
-
-        verify(oneginiSdk).setPinNotificationObserver(any())
+        verify(oneginiSdk).setCreatePinEventEmitter(any())
```

Comment:
When tried to run the tests I noticed that imports for OneginiWrapperErrors are incorrect

Reply:
We will fix this with the ORS-72 merge